projects/dxilconv/lib/DxilConvPasses/DxilCleanup.cpp

```diff
@@ -1265,6 +1266,7 @@ Value *DxilCleanup::CastValue(Value *pValue, Type *pToType, Instruction *pOrigIn
     ArgTypes[2] = Type::getInt64Ty(*m_pCtx);
   } else {
     IFT(DXC_E_OPTIMIZATION_FAILED);
+    OpCode = DXIL::OpCode::NumOpCodes;
```

Comment:
Nit: Since the line above will unconditionally throw, this will never be reached. I know the error you're trying to avoid, but you could also avoid it by initializing the opcode on the first line it's declared.